backend/DatabaseScripts/SqlScripts/0039_CreateView_graphql_contracts_view.sql

```diff
@@ -0,0 +1,108 @@
+﻿ALTER TABLE graphql_transaction_events
+	ALTER COLUMN event SET DATA type jsonb;
+CREATE INDEX event_tag on graphql_transaction_events (((event->>'tag')::INTEGER));
+CREATE INDEX event_contract_init_contract_address ON graphql_transaction_events (((event->'data'->>'ContractAddress'))) WHERE (event->>'tag')::INTEGER = 16;
+CREATE INDEX event_contract_upgrade_contract_address ON graphql_transaction_events (((event->'data'->>'ContractAddress'))) WHERE (event->>'tag')::INTEGER = 36;
+CREATE INDEX event_contract_update_contract_address ON graphql_transaction_events (((event->'data'->>'ContractAddress'))) WHERE (event->>'tag')::INTEGER = 18;
+CREATE INDEX event_contract_transfer_from_contract_address ON graphql_transaction_events (((event->'data'->'From'->>'data'))) WHERE (event->>'tag')::Integer = 1 AND (event->'data'->'From'->>'tag')::Integer = 2;
+CREATE OR REPLACE VIEW graphql_contracts_view 
```

Comment:
Since this is a view and hence a query all this is run each time.
https://www.postgresql.org/docs/current/sql-createview.html

I would prefer making this logic once in a table to get up-to-date once.